german.html

```diff
@@ -0,0 +1,115 @@
+<html>
+<head>
+  <meta charset="UTF-8">
+  <script language="javascript" type="text/javascript" src="https://cdn.jsdelivr.net/npm/p5@1.0.0/lib/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="SocialDistancingSimulator.js"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="container">
+  <h1>'Social distancing' Simulator</h1>
+
+  <p>Mit diesem Simulator kannst du den Effekt von 'social distancing' auf die
```

Comment:
"du"? Is this idiomatic German usage? In Slovene you'd definitely use the polite form.

Reply:
You don't call your boss "Du" on first day of work like "you" in English, but I wouldn't expect the formal "Sie" on a small infotainment page. Although one could even capitalize the Du. Otherwise, "kannst du" becomes "können Sie."